### PR TITLE
Add class in control flow exercises

### PR DESCRIPTION
**Pull request by MarianBastiurea**
Hope to have no mistakes

### CODE REVIEW

File: src/main/java/org/example/c03/controlFlow/NumberOfDaysInMonth.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ Be careful about leap years they have 29 days in month 2 (February).
+You should check if the year is a leap year using the method isLeapYear described above.
+*/
+
+public class NumberOfDaysInMonth {
+    public static int getDaysInMonth(int month, int year) {
+        int result;
+        if (month < 1 || month > 12 || year <= 1 || year >= 9999) {
+            result = -1;
+            return result;
```
Comment (andreiursudev) [inline src/main/java/org/example/c03/controlFlow/NumberOfDaysInMonth.java]: simplify to  return -1;

File: src/main/java/org/example/c03/controlFlow/NumberOfDaysInMonth.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        return result;
+    }
+
+    public static boolean isLeapYear(int year) {
+        boolean leapYear;
+        if (year < 1 || year > 9999) {
+            leapYear = false;
+        } else if (year % 4 == 0 && year % 100 != 0) {
+            leapYear = true;
+        } else leapYear = year % 100 == 0 && year % 400 == 0;
```
Comment (andreiursudev) [inline src/main/java/org/example/c03/controlFlow/NumberOfDaysInMonth.java]: remove year % 100 == 0. Is not needed. If year % 400 == 0 the automatically year % 100 == 0

File: src/test/java/org/example/c03/ontrolFlow/DayOfTheWeekTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    @Test
+    public void given0ShouldReturnSunday() {
+        int day = 0;
+        String myDay = "Sunday";
+        String result = DayOfTheWeek.printDayOfTheWeek(day);
+        assertEquals(myDay, result);
+    }
+
+    @Test
+    public void given7ShouldReturnSunday() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c03/ontrolFlow/DayOfTheWeekTest.java]: Incorrect test name. It returns "Invalid day".

File: src/test/java/org/example/c03/ontrolFlow/PrimeNumberCounterTest.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.example.c03.controlFlow.PrimeNumberCounter;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class PrimeNumberCounterTest {
+    @Test
+    public void given3and9ShouldReturn3PrimeNumber() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c03/ontrolFlow/PrimeNumberCounterTest.java]: Fix name : givenLower3Upper9ThenReturn3PrimeNumbers

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/c03/controlFlow/NumberOfDaysInMonth.java (hunk 1 above)
Comment (MarianBastiurea) [reply src/main/java/org/example/c03/controlFlow/NumberOfDaysInMonth.java]: done

File: src/main/java/org/example/c03/controlFlow/NumberOfDaysInMonth.java (hunk 2 above)
Comment (MarianBastiurea) [reply src/main/java/org/example/c03/controlFlow/NumberOfDaysInMonth.java]: was a test condition.  Tim did in same way.
Change it

File: src/test/java/org/example/c03/ontrolFlow/DayOfTheWeekTest.java (hunk 3 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c03/ontrolFlow/DayOfTheWeekTest.java]: Done

File: src/test/java/org/example/c03/ontrolFlow/PrimeNumberCounterTest.java (hunk 4 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c03/ontrolFlow/PrimeNumberCounterTest.java]: Done

Comment (andreiursudev) [reply src/test/java/org/example/c03/ontrolFlow/PrimeNumberCounterTest.java]: PrimeNumbers not PrimeNumber

Comment (MarianBastiurea) [reply src/test/java/org/example/c03/ontrolFlow/PrimeNumberCounterTest.java]: done

Comment (andreiursudev) [reply src/test/java/org/example/c03/ontrolFlow/PrimeNumberCounterTest.java]: I was referring to the test name. Its still not good

Comment (MarianBastiurea) [reply src/test/java/org/example/c03/ontrolFlow/PrimeNumberCounterTest.java]: done. thanks

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**